### PR TITLE
test: add common test methods for RBAC

### PR DESCRIPTION
**Pull request by Parthvi12**
Adds common test methods that are to be used in RBAC and fix flaky test

### CODE REVIEW

File: app/client/cypress/support/index.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ import "./queryCommands";
 import "./widgetCommands";
 import "./themeCommands";
 import "./AdminSettingsCommands";
+import "./RBACCommands";
```
Comment (ankitakinger) [inline app/client/cypress/support/index.js:33]: this RBACCommands file won't exist on CE right? This change is not required on CE.

Comment (Parthvi12) [reply app/client/cypress/support/index.js:33]: But we will use it in EE , adding it here to avoid merge conflicts in future.

Comment (ankitakinger) [reply app/client/cypress/support/index.js:33]: But this file doesn't exist in CE. This import line will lead to an error. We have done the same for OIDCCommands & SAMLCommands as well.

Comment (Parthvi12) [reply app/client/cypress/support/index.js:33]: Okay removing it, thanks.

**Review by ankitakinger — APPROVED** [review]

**Pull request merged**